lib/dor/services/client/object.rb

```diff
@@ -106,6 +106,18 @@ def update_doi_metadata
           raise_exception_based_on_response!(resp)
         end
 
+        # Update the ORCID Works
```

Comment:
Will this method be updating multiple ORCID works?

Reply:
So, that's really a metaphysical question as to what constitutes a "work" in the orcid system. However, I have been using the singular elsewhere, so fixing here.